src/components/FeatureViews/Accounts/TabComponents/Schedule.js

```diff
@@ -8,10 +10,37 @@ class Schedule extends Component {
         };
     }
 
+    // get instructors schdeule and parses it into one array of objcts 
+    getInstructorSchedule = () => {
+        let schedule = this.props.work_hours
+        let key = Object.keys(this.props.work_hours)
+
+        let instructorsSchedule = key.map((iKey) => {
+            return schedule[iKey]
+        })
+        let finalInstructorSchedule = []
+        instructorsSchedule.forEach((iList) => {
+            finalInstructorSchedule = finalInstructorSchedule.concat(Object.values(iList))
+        })
```

Comment:
None of this is used - remove this code if you don't need `finalInstructorSchedule`.